ui/components/app/transaction-decoding/components/ui/accreditation/index.scss

```diff
@@ -0,0 +1,27 @@
+.accreditation {
+  display: flex;
+  font-size: 12px;
```

Comment:
Can we change this to `0.75rem` instead of `12px`?  We try to avoid pixel sizing with fonts.